sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/ModelConverter.java

```diff
@@ -0,0 +1,39 @@
+package com.azure.digitaltwins.core.models;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.time.OffsetDateTime;
+import java.util.Map;
+
+/**
+ * Creates publicly facing object types
+ */
+public class ModelConverter {
+    public static DigitalTwinModelData convertToDigitalTwinModelData(
+        String id,
+        Object model,
+        OffsetDateTime uploadTime,
+        Map<String, String> description,
+        Map<String, String> displayName,
+        Boolean isDecommissioned)
+    {
+        String modelStringValue = null;
+
+        if (model!= null){
+            try {
+                modelStringValue = new ObjectMapper().writeValueAsString(model);
+            } catch (JsonProcessingException e) {
+                e.printStackTrace();
```

Comment:
Should we thrown an exception here? An exception here indicates that service sent us incorrectly formatted data, we should surface this exception up.